.env.example

```diff
@@ -4,4 +4,6 @@ PG_DATABASE=pgdatabase
 PG_HOST=localhost
 PG_PORT=5432
 CONSUMER_SECRET=JEsWifMZQc76z7244aAhiXyMltfo4z9Jd
-CONSUMER_KEY=2aH0sk2CCAD9bpBTfYzN9OXrQWnkD80
\ No newline at end of file
+CONSUMER_KEY=2aH0sk2CCAD9bpBTfYzN9OXrQWnkD80
+APP_SECRET='sasdfsdfsdfdsf' #can be any random string, hard to guess
+FRONTEND_URL='http://localhost:8000'
```

Comment:
Would it make sense to use `http://localhost:8889` here as an example since that's the port `npm start` hosts the service on?

Reply:
This is meant to be the URL that is serving the `frontend`, so not sure what port that would default to..